### PR TITLE
add basic html strucutre with text but missing images, links  etc.

### PR DESCRIPTION
**Pull request by alper92**
Hi there, can you please check my code...

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+          <li>Travelling</li>
+          <li>Food</li>
+          <li>New Cultures</li>
+          <li>Good Music</li>
+        </ul>
+      </div>
+    </section>
+    <section id="contact">
+      <h2>Contact:</h2>
+      <p>Number: <span>0175 7777777</span></p>
```
Comment (ninaassmann) [inline index.html:37]: you can make a clickable link, so u can automatically call oder mail
<a href="tel:123456789">numer xxx</a>
<a href="mailto:xyz@blabla.de">link text here</a>

**Review by ninaassmann — CHANGES_REQUESTED** [review]
looks good, but recommend using more than one commit for better structure

File: index.html (hunk 1 above)
Comment (alper92) [reply index.html:37]: Thanks for the hint...

File: index.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      <p>E-Mail: <span>alperkuertuel@mail.de</span></p>
```
Comment (ninaassmann) [inline index.html:38]: <p>E-Mail: <span>alperkuertuel@mail.de</span></p>

better do this:
<p>E-Mail: <a href="mailto:alperkuertuel@mail.de">alperkuertuel@mail.de</a></p>

**Review by ninaassmann — CHANGES_REQUESTED** [review]
one more hint

File: index.html (hunk 2 above)
Comment (alper92) [reply index.html]: I added already a <span>-tag  for indentation of the elements with giving the span a class...  but yes if we look at with first sight it it seems to be useless

**Pull request merged**